Dockerfile

```diff
@@ -40,27 +40,38 @@ FROM builder as testing
 
 WORKDIR /workspace
 
-RUN apk add rsync curl tar
+RUN apk add curl tar
 
 # These two steps follow the Kubebuilder envtest playbook https://book.kubebuilder.io/reference/envtest.html
 ARG K8S_VERSION=1.23.3
 RUN curl -sSLo envtest-bins.tar.gz "https://go.kubebuilder.io/test-tools/${K8S_VERSION}/$(go env GOOS)/$(go env GOARCH)" \
     && mkdir /usr/local/kubebuilder \
     && tar -C /usr/local/kubebuilder --strip-components=1 -zvxf envtest-bins.tar.gz
 
-
 COPY config/ config/
 COPY initiateContainerTest.sh .
 
 ENTRYPOINT [ "sh", "initiateContainerTest.sh" ]
 
 ###############################################################################
-FROM alpine:latest
+FROM ghcr.io/nearnodeflash/nnf-mfu:latest
+
+RUN apt update
+
+RUN apt install -y openmpi-bin
+
+# TODO Remove this
+RUN apt install -y bash
 
-RUN apk add rsync
+# The following lines are from the mpiFileUtils (nnf-mfu) Dockerfile; 
+# do not change them unless you know what it is you are doing
+ARG port=2222
+RUN sed -i "s/[ #]\(.*StrictHostKeyChecking \).*/ \1no/g" /etc/ssh/ssh_config \
+    && sed -i "s/[ #]\(.*Port \).*/ \1$port/g" /etc/ssh/ssh_config \
+    && echo "    UserKnownHostsFile /dev/null" >> /etc/ssh/ssh_config
```

Comment:
If you're using the `nnf-mfu` image above, why do these commands again?

Reply:
Using the nnf-mfu is a stop gap because I can't get the nnf-dm container to talk to the nnf-mfu container.